packages/e2e-cypress/src/install.js

```diff
@@ -4,24 +4,77 @@
  * API: https://github.com/quasarframework/quasar/blob/master/app/lib/app-extension/InstallAPI.js
  */
 
+/**
+ * Performs a deep merge of objects and returns new object. Does not modify
+ * objects (immutable) and merges arrays via concatenation.
+ * based on https://stackoverflow.com/a/49798508
+ *
+ * @param {...object} sources - Objects to merge
+ * @returns {object} New object with merged key/values
+ */
+
+function __mergeDeep(...sources) {
```

Comment:
I'm pretty sure I've seen this `__mergeDeep` code in [another place](https://github.com/quasarframework/quasar-testing/search?q=https%3A%2F%2Fstackoverflow.com%2Fa%2F49798508&unscoped_q=https%3A%2F%2Fstackoverflow.com%2Fa%2F49798508) in the code.  Should it be moved to it's own utility to minimize duplication and maintenance effort?

Reply:
Given the strange nature of "monorepo of app-extensions", I'm not sure dedupe is possible.
I mean, it is possible if the file is put in a standalone library (or we add a dependency on an external one already providing it), but otherwise I think it's not (but I'm not sure).
Gonna check if some lightweight library have this helper

Reply:
Will leave dedupe to a later PR, as this is already way too big